src/Host.cpp

```diff
@@ -1502,9 +1502,9 @@ void Host::processGMCPDiscordInfo(const QJsonObject& discordInfo)
 
     if (hasInvite) {
         if (hasCustomAppID) {
-            qDebug() << "Game using a custom Discord server. Invite URL: " << inviteUrl.toString();
+            qDebug() << "Game using a custom Discord server. Invite URL:" << inviteUrl.toString();
```

Comment:
This is intended as two spaces were produced otherwise (qDebug() inserts spaces unless you use `.nospace()`)